
<!-- Starts up everything needed to run the example listener/talker Python nodes. -->
<launch>

  <arg name="lta3_id" default="lta0"/>



<group ns="$(arg lta3_id)">


  <node pkg="lta3" type="statemux2.py" name="StateMUX">
    <param name="lta3_id" type = "string" value="$(arg lta3_id)"/>
    <param name="selector_type" type = "string" value="vicon_nofilter"/>
  </node>


  <node pkg="lta3" type="controller.py" name="Controller" output="screen">
    <param name="controller_type" type = "string" value="MAV1_PID"/>
    <param name="update_rate" value="10"/>
  </node>


  <node pkg="lta3" type="IK.py" name="IK">
    <param name="blimp_type" type = "string" value="MAV1"/>
  </node>

  <node pkg="lta3" type="driver_ref.py" name="Driver_Ref" output="screen">
    <param name="blimp_type" type = "string" value="MAV1"/>
    <param name="lta3_id" type = "string" value="$(arg lta3_id)"/>
  </node>

  <node pkg="lta3" type="driver_state.py" name="Driver_State">
    <param name="selector_type" type = "string" value="ping_altitude"/>
    <param name="lta3_id" type = "string" value="$(arg lta3_id)"/>
  </node>


  <node pkg="lta3" type="manualwaypoints.py" name="waypoints" output="screen" launch-prefix="xterm -e">
  </node>

</group>

</launch>
